cli/cephadm/cephadm.py

```diff
@@ -40,3 +41,15 @@ def rm_cluster(self, fsid, zap_osds=True, force=True):
             cmd += " --force"
 
         return self.execute(sudo=True, long_running=True, cmd=cmd)
+
+    def prepare_host(self, **kw):
```

Comment:
If prepare-host command expects only one parameter then no need to use **kw, use `hostname` param

Reply:
@vamahaja 
Since it is a optional parameter we have added **kw, Present test case not required hostname

Reply:
ok